DataFormats/Common/interface/ThinnedRefSet.h

```diff
@@ -0,0 +1,50 @@
+#ifndef DataFormats_Common_ThinnedRefSet_h
+#define DataFormats_Common_ThinnedRefSet_h
+
+#include "DataFormats/Common/interface/Handle.h"
+#include "DataFormats/Common/interface/Ref.h"
+#include "DataFormats/Common/interface/RefItemGet.h"
+
+#include <unordered_set>
+
+namespace edm {
+  class EDProductGetter;
+
```

Comment:
Does this class exist only to support unit tests? A comment saying that would be helpful. Or an explanation of its intended use.

Reply:
Added comment explaining the intended use.